regulatory/Makefile

```diff
@@ -0,0 +1,39 @@
+SHELL := /bin/bash -o pipefail -e
+.SUFFIXES:
+.SECONDARY:
+.DELETE_ON_ERROR:
+
+SYSTEM_CONFIG := data/system.yml
+DATA_FILES := $(wildcard data/*.yml)
+TEMPLATES := $(wildcard templates/*.md)
+RELEASE_DOCUMENTS := $(patsubst documents/%.md,release/%.md,$(wildcard documents/*.md))
+RELEASE_PDFS := $(patsubst documents/%.md,release/%.pdf,$(wildcard documents/*.md))
+RELEASE_JSONS := $(patsubst documents/%.md,release/%.pdf,$(wildcard documents/*.md))
+
+all: $(RELEASE_DOCUMENTS)
+
+pdfs: $(RELEASE_PDFS)
+
+jsons: $(RELEASE_JSONS)
+
+.PHONY:
+github_issues: $(SYSTEM_CONFIG)
+	rdm pull $<
+
+release/%.md: documents/%.md $(DATA_FILES) $(TEMPLATES)
+	@mkdir -p $(@D)
+	rdm render $< $(DATA_FILES) > $@
+
+tmp/%.tex: release/%.md
+	@mkdir -p $(@D)
+	rdm tex $< $(DATA_FILES) > $@
+
+release/%.pdf: tmp/%.pdf
+	cp $< $@
+
+tmp/%.pdf: tmp/%.tex
+	latexmk -pdf -jobname="./$(basename $@)" $<
```

Comment:
I think devs might be confused by a directory  named `tmp` sitting around as a result of the build process.  "Shouldn't the build have nuked this directory when it was done with it?  Why didn't it use /tmp?"  Perhaps all that needs to be done is to call this directory "build" instead.

Reply:
tmp is intermediate files generated during the make process. They should be nuked automatically. However, if the process is stopped there may be tmp files generated, so it is just an extra safety precaution throwing it in the .gitignore

Reply:
@jlebar what makes you think `/tmp` would be better than a local directory? I figured a local directory would be better so you can investigate the intermediate files more easily if you wanted to. I wouldn't be opposed to renaming the directory `build` instead of `tmp`.

Note that it is nice to keep the files around for a bit so that you can take advantage of incremental complication.